sdk/keyvault/azure-security-keyvault-administration/CMakeLists.txt

```diff
@@ -39,9 +39,9 @@ if(FETCH_SOURCE_DEPS)
         add_subdirectory(${oneFolder})
     endforeach()
 elseif(NOT AZ_ALL_LIBRARIES)
-  find_package(azure-core-cpp "1.5.0" CONFIG QUIET)
+  find_package(azure-core-cpp "1.8.0" CONFIG QUIET)
   if(NOT azure-core-cpp_FOUND)
-    find_package(azure-core-cpp "1.5.0" REQUIRED)
+    find_package(azure-core-cpp "1.8.0" REQUIRED)
```

Comment:
Sorry for the initially bad suggestion, let's just drop the version in this file only. The other two containing 1.8.0. are good.

Reply:
It would work after we release Core, but still we should not do it in this file.

Reply:
so it will work after tomorrow, let's leave it there then , i'm merging it on tuesday anyway

Reply:
besides , all the other packages use the same syntax, where they specify the version number in the find(...)